### DISCUSSION TITLE
Metrics per dashboard

### DISCUSSION THREAD

**Discussion by mforgues** [unfor19/frigga#10] [General]
Hi,

I noticed something while trying out the frigga metric tool.

Seems like the breakdown of the metrics per dashboard is off a little.

Like the detected metrics of the first dashboard are also contained in the subsequent one and etc.

Overall metric list is good but also try to see in the python code to find the issue but I am a beginner...

Example of what I mean :

  "dashboards": {
    "baremetal_detailed_v1": {
      "gnet_id": "null",
      "metrics": [
        "idmgroup",
        "instance",
        "label_values",
        "power_supply",
        "processor",
        "redfish_chassis_fan_health",
        "redfish_chassis_fan_rpm_percentage",
        "redfish_chassis_fan_state",
        "redfish_chassis_network_adapter_health_state",
        "redfish_chassis_network_adapter_state",
        "redfish_chassis_network_port_health_state",
        "redfish_chassis_network_port_state",
        "redfish_chassis_power_average_consumed_watts",
        "redfish_chassis_power_powersupply_health",
        "redfish_chassis_power_powersupply_power_capacity_watts",
        "redfish_chassis_power_powersupply_state",
        "redfish_chassis_power_voltage_volts",
        "redfish_chassis_temperature_celsius",
        "redfish_chassis_temperature_sensor_state",
        "redfish_system_health_state",
        "redfish_system_memory_capacity",
        "redfish_system_memory_health_state",
        "redfish_system_memory_state",
        "redfish_system_network_interface_health_state",
        "redfish_system_network_interface_state",
        "redfish_system_processor_health_state",
        "redfish_system_processor_state",
        "redfish_system_processor_total_cores",
        "redfish_system_processor_total_threads",
        "redfish_system_storage_drive_capacity",
        "redfish_system_storage_drive_state",
        "redfish_system_storage_volume_capacity",
        "redfish_system_storage_volume_state",
        "sensor",
        "volume"
      ],
      "num_metrics": 35
    },
    "cadvisor": {
      "gnet_id": "null",
      "metrics": [
        "cadvisor_version_info",
        "container_cpu_usage_seconds_total",
        "container_last_seen",
        "container_memory_max_usage_bytes",
        "container_memory_rss",
        "container_memory_usage_bytes",
        "container_network_receive_bytes_total",
        "container_network_transmit_bytes_total",
        "container_spec_memory_limit_bytes",
        "idmgroup",
        "instance",
        "label_values",
        "power_supply",
        "processor",
        "redfish_chassis_fan_health",
        "redfish_chassis_fan_rpm_percentage",
        "redfish_chassis_fan_state",
        "redfish_chassis_network_adapter_health_state",
        "redfish_chassis_network_adapter_state",
        "redfish_chassis_network_port_health_state",
        "redfish_chassis_network_port_state",
        "redfish_chassis_power_average_consumed_watts",
        "redfish_chassis_power_powersupply_health",
        "redfish_chassis_power_powersupply_power_capacity_watts",
        "redfish_chassis_power_powersupply_state",
        "redfish_chassis_power_voltage_volts",
        "redfish_chassis_temperature_celsius",
        "redfish_chassis_temperature_sensor_state",
        "redfish_system_health_state",
        "redfish_system_memory_capacity",
        "redfish_system_memory_health_state",
        "redfish_system_memory_state",
        "redfish_system_network_interface_health_state",
        "redfish_system_network_interface_state",
        "redfish_system_processor_health_state",
        "redfish_system_processor_state",
        "redfish_system_processor_total_cores",
        "redfish_system_processor_total_threads",
        "redfish_system_storage_drive_capacity",
        "redfish_system_storage_drive_state",
        "redfish_system_storage_volume_capacity",
        "redfish_system_storage_volume_state",
        "sensor",
        "volume"
      ],
      "num_metrics": 44
    },

Mathieu

**Comment by unfor19** [comment]
Hi (again) Mathieu, I didn't get a notification for this thread, I'll make sure I'm subscribed so I won't miss msgs, luckily you've created an issue and it was resolved #11 :)
